src/components/tipRecords/TipRecord.vue

```diff
@@ -70,6 +71,9 @@ export default {
       if (!this.isPreviewToBeVisualized(this.tip)) return '';
       return this.tip.preview.description ? this.tip.preview.description : this.tip.preview.title;
     },
+    tipPreviewImage() {
+      return this.isPreviewToBeVisualized(this.tip) ? `${BACKEND_URL}${this.tip.preview.image}` : '';
```

Comment:
maybe build url in backendinstance, to have all urls summarized there

Reply:
I guess we could do @mradkov whats your opinion?

Reply:
Yes, I agree with @thepiwo, if we are keeping the url there having all urls summarized there is reasonable.